CommonLibSF/cmake/make-sourcelist.ps1

```diff
@@ -0,0 +1,64 @@
+#Requires -Version 5
+
+# args
+param (
+    [string]$PathIn,
+    [string]$PathOut
+)
+
+$ErrorActionPreference = "Stop"
+
+$SourceExt = @('.asm', '.c', '.cc', '.cpp', '.cxx', '.h', '.hpp', '.hxx', 'inc', '.inl', '.ixx')
```

Comment:
Commonly in addition to ixx modules are named cppm or mxx.

Reply:
Added. Currently I have stl modules disabled for this PR, note that in the future when we do anything with modules.
c7eee664ab9252bf59d2a4f12069dc90200a92e2